WcaOnRails/app/views/polls/index.html.erb

```diff
@@ -0,0 +1,69 @@
+<% provide(:title, 'Polls') %>
+
+<div class="container">
+  <h1><%= yield(:title) %></h1>
+
+  <% can_create = current_user.can_create_poll? %>
+  <% if can_create %>
+    <%= link_to "New poll", new_poll_path, class: "new-poll" %>
+  <% end %>
+
+  <% if @polls.empty? %>
+    <div class="alert alert-warning">There are currently no polls.</div>
+  <% else %>
+    <table class="table polls-list">
+      <%# @polls.each do |poll| %>
+      <h4>Open polls</h4>
+      <% @openPolls.each do |poll| %>
+        <% if can_create && !poll.confirmed %>
+          <% unconfirmed = true %>
+        <% else %>
+          <% unconfirmed = false %>
+        <% end %>
+        <tr class="<%= unconfirmed ? "danger" : "" %>" data-toggle="tooltip" data-placement="top" data-container="body" title="<%= unconfirmed ? "This poll is not confirmed" : "" %>">
+          <td><%=md poll.question %></td>
+          <td>
+            <% if poll.user_already_voted?(current_user) %>
+              <span class="glyphicon glyphicon-ok-sign" aria-hidden="true" data-toggle="tooltip" data-placement="top" data-container="body" title="You have already voted in this poll"></span>
+            <% end %>
+          </td>
+          <td>
+            <% if poll.user_already_voted?(current_user) %>
+              <%= link_to "Edit vote", polls_vote_path(poll) %>
+            <% else %>
+              <%= link_to "Vote", polls_vote_path(poll) %>
+            <% end %>
+          </td>
+          <% if current_user.can_create_poll? %>
+            <td><%= link_to "Edit", edit_poll_path(poll) %></td>
+          <% end %>
+        </tr>
+      <% end %>
+    </table>
+
+    <h4>Closed polls</h4>
+    <table class="table polls-list">
```

Comment:
This table looks like a copy of the previous table, lets pull them out into a partial.